tests/tests_fabric/test_connector.py

```diff
@@ -497,9 +491,9 @@ def test_strategy_choice_ddp_spawn_cpu():
 @RunIf(skip_windows=True)
 @mock.patch("lightning.fabric.connector._IS_INTERACTIVE", True)
 def test_strategy_choice_ddp_fork_in_interactive():
-    """Test that when accelerator and strategy are unspecified, the connector chooses DDP Fork in interactive
-    environments by default."""
```

Comment:
But I wanted to test specifically what was here. Sure, your case is valid too, but this configuration should stay I think.

Reply:
To test it with "auto" accelerator, we need to mock each environment availability, as done for `test_connector_auto_selection`. Is this what you are proposing?

Reply:
You can simply add mps_count_0 and cuda_count_0 as test arguments. That should work no?

Reply:
It would fail if you run the test in a TPU/HPU/IPU machine. Hence the complexity in `test_connector_auto_selection`

Reply:
I added assertions to it